styles/all/template/js/collapsiblecategories.js

```diff
@@ -0,0 +1,46 @@
+(function ($) {  // Avoid conflicts with other libraries
+
+	'use strict';
+
+	$('a.category[data-id] + .forabg').each(function () {
+		var $this = $(this),
+			id = $this.prev().attr('data-id'),
+			hidden = $this.prev().attr('data-hidden'),
```

Comment:
Using `prev()` could potentially cause problems with other extensions that insert elements. Better use a find/child (and reuse it rather than finding it twice).